### PR TITLE
paypal-initial_test

### PR DESCRIPTION
**Pull request by msk4862**
For #104 paypal-feature

I've used 'react-async-script-loader' package.

You need to first add your sandbox or production paypal CLIENT ID in pages/donate.js for testing.

### CODE REVIEW

**Comment by stripedpajamas** [comment]
Hi @msk4862 👋 

Thanks so much for this PR 👍 
I'm wondering if the method that @joelwass commented in #104 would work or if you tried it and ran into issues? From the comment in that issue, I would expect that we can put those `<input>` tags into the donate form and that would create a button that is linked to the TeacherFund PayPal account automatically. If that works, I'd rather do it that way as it would be easier to maintain (and easier for future contributors to understand).

**Comment by msk4862** [comment]
Okay @stripedpajamas 
I'll do the changes 👍👍

**Comment by msk4862** [comment]
Why my single quotes are converted into double quotes?😕

**Comment by msk4862** [comment]
@stripedpajamas @joelwass can you please review the changes

File: pages/donate.js [hunk 1]
```diff
@@ -31,6 +31,17 @@ class Donate extends Component {
                 </Elements>
               </div>
             </StripeProvider>
+            <div className='flex flex-column w-120 m-auto'>
+              <div className='ttu v-mid m-auto'>
```
[the remark this answers is not in the document]
Comment (msk4862) [reply pages/donate.js]: @joelwass <div className='ttu v-mid m-auto'> 
in this?

[the remark this answers is not in the document]
Comment (joelwass) [reply pages/donate.js]: yup! add tf-lato in there to style the text

[the remark this answers is not in the document]
Comment (msk4862) [reply pages/donate.js]: @joelwass done.

**Comment by msk4862** [comment]
@joelwass  is it done?

**Comment by msk4862** [comment]
@joelwass @stripedpajamas I've fixed the deployment failure, please check

**Pull request merged**